<Action>
  <Thought>Failed to generate a valid action due to: {{ error }}</Thought>
  <Error>
    <Message>{{ error }}</Message>
  </Error>
  <Code><![CDATA[
import asyncio
async def main():
    print("Error: Action generation failed")
  ]]></Code>
</Action>